Documentation for dotnet-trace report topN (#27062)
dotnet/docs: docs/core/diagnostics/dotnet-trace.md

dotnet-trace list-profiles [-h|--help]
```

## dotnet-trace report

> Generates a report into stdout from a previously generated trace.

Reviewer: ```suggestion
Creates a report into stdout from a previously generated trace.
```

Just so generate isn't used twice in the sentence.

### Synopsis

```console
dotnet-trace report [-h|--help] <tracefile> [command]
```

### Arguments

- **`<tracefile>`**

  The file path for the trace being analyzed.

### Commands

#### dotnet-trace report topN

> Finds the top N methods on the callstack the longest.

Reviewer: ```suggestion
Finds the top N methods that have been on the callstack the longest.
```

##### Synopsis

```console
dotnet-trace report <tracefile> topN [-n|--number <n>] [--inclusive] [-h|--help]
```

##### Options

- **`-n|--number <n>`**

Gives the top N methods in the callstack.

- **`--inclusive`**

> Output the topN methods based on inclusive time. If not specified, exclusive time is used by default.

Reviewer: ```suggestion
Output the top N methods based on inclusive time. If not specified, exclusive time is used by default.
```

Might be worth explaining what inclusive means?

dotnet-trace list-profiles [-h|--help]
```

## dotnet-trace report

Creates a report into stdout from a previously generated trace.

### Synopsis

```console
dotnet-trace report [-h|--help] <tracefile> [command]
```

### Arguments

- **`<tracefile>`**

  The file path for the trace being analyzed.

### Commands

#### dotnet-trace report topN

Finds the top N methods that have been on the callstack the longest.

##### Synopsis

```console
dotnet-trace report <tracefile> topN [-n|--number <n>] [--inclusive] [-v|--verbose] [-h|--help]
```

##### Options

- **`-n|--number <n>`**

Gives the top N methods on the callstack.

- **`--inclusive`**

> Output the top N methods based on [inclusive](https://docs.microsoft.com/visualstudio/profiling/understanding-sampling-data-values?view=vs-2017) time. If not specified, exclusive time is used by default.

Reviewer: ```suggestion
Output the top N methods based on [inclusive](/visualstudio/profiling/understanding-sampling-data-values) time. If not specified, exclusive time is used by default.
```
